### ISSUE TITLE
ask for help：Error in reading symmetry

### ISSUE THREAD

**Issue by lyonworton** [adrybakov/rad-tools#19]
Hello, I would like to report to you a problem encountered with rad-tools reading the output file of TB2J. When I use rad-plot-tb2j-magnons.py to read the TB2J output file exchange.out file, it always shows that the symmetry of the structure read is not what I want, but I double check the information in the exchange.out file, for example, the file shows that it's a HEX structure, but it always reads ORCC or MCL, what is wrong?  I really hope you can give me guidance and help，Thanks！
[result.zip](https://github.com/adrybakov/rad-tools/files/15417075/result.zip)

**Comment by adrybakov** [comment]
Hello! 

Thank you for  the interest in the package!

I will take a look in the following couple of days.

Could you tell which version do you use?

The following command should display the version (python or python3 - depending on your situation.):
```
python3 -c "exec('from radtools import __version__\nprint(__version__)')"
```

Does it help if you'll pass `-bt HEX` option? (if it crashes - please attach error output from console)

Best,
Andrey

**Comment by lyonworton** [comment]
Hello, thank you for your quick reply, I checked and found that I am using radtools version 0.8.13. The example in the screenshot is using the file in the zip file I sent before, and the symmetry is supposed to be HEX, but it can't be recognized, and I don't know if there is something wrong with my operation or what is going on, so I would like to ask you for some advice.
![Screenshot from 2024-05-24 15-51-23](https://github.com/adrybakov/rad-tools/assets/112809823/ec130df5-2417-45a4-b572-e1c0ad98e8b8)

**Comment by adrybakov** [comment]
Thank you for your replay!

It seems to be a bug on rad-tools side. I will try to solve it next week.

Best,
Andrey

**Comment by adrybakov** [comment]
Hi!

The problem seems to be resolved in the release 0.9.0.

**Comment by adrybakov** [comment]
Hi!

The problem seems to be resolved in the release 0.9.0

Please update it with 

```
pip install rad-tools --upgrade
```

and confirm whether the problem solved.

NOTE: in this release the naming convention for scripts was changed: ".py" was removed. I.e. the execution line should look as follows:

```
rad-plot-tb2j-magnons -if ...
```

Best,
Andrey
